issuer/tests/issue_credentials.rs

```diff
@@ -658,7 +671,15 @@ fn should_issue_vc_and_validate_e2e() -> Result<(), CallError> {
             env.time().duration_since(UNIX_EPOCH).unwrap().as_nanos(),
         )
         .expect("credential verification failed");
-        let vc_claims = claims.vc().expect("missing VC claims");
+        let vc_claims = claims
+            .custom()
+            .expect("missing custom claims in JWT claims")
+            .as_object()
+            .expect("malformed custom claims in JWT claims")
+            .get("vc")
+            .expect("missing vc claims in JWT custom claims")
+            .as_object()
+            .expect("malformed vc claims in JWT custom claims");
         validate_claims_match_spec(vc_claims, &spec).expect("Clam validation failed");
```

Comment:
fly-by catch: typo in the error message: "Clam" -> "Claims"

Reply:
Thanks!